lib/brains/serializers.py

```diff
@@ -27,6 +28,24 @@ def data(self):
         }
 
 
+class LocalReceipt(serializers.Serializer):
+
+    """
+    Serialize enough information to send the user a receipt in one go.
+    """
+
+    def __init__(self, **kw):
```

Comment:
if this were ``def __init__(self, paymethod, subscription, transaction)`` then you could still pass them in as keywords. It would read a bit cleaner.